.github/workflows/release.yml

```
@@ -23,19 +23,17 @@ jobs:
+        uses: ./.github/actions/install-deps
 
-      - name: Read .nvmrc
-        run: |
-          NODE_VERSION=$(cat .nvmrc)
-          echo "NODE_VERSION=$NODE_VERSION" >> $GITHUB_OUTPUT
-        id: nvm
-
-      - uses: actions/setup-node@v5
+      - uses: actions/setup-node@v6
         with:
           node-version: ${{ steps.nvm.outputs.NODE_VERSION }}
```

actionlint (1.7.9)

property "nvm" is not defined in object type {}

(expression)